docs/operator-config.asciidoc

```diff
@@ -26,7 +26,8 @@ ECK can be configured using either command line flags or environment variables.
 |operator-namespace |"" |Namespace the operator runs in. Required.
 |operator-roles |all |Roles this operator should assume. Valid values are `namespace`, `global`, `webhook` or `all`. Accepts multiple comma separated values.
 |webhook-pods-label |"" |Label used to select pods running the webhook server.
-|webhook-secret |"" | Kubernetes secret containing the webhook certificates.
+|webhook-secret |"" | K8s secret mounted into the path designated by webhook-cert-dir to be used for webhook certificates.
+|webhook-cert-dir |"{TempDir}/k8s-webhook-server/serving-certs" |Path to the directory that contains the webhook server key and certificate.
```

Comment:
Out of curiosity, I wonder what can be the default value of "TempDir" other than `/tmp`?

Reply:
https://golang.org/pkg/os/#TempDir
> On Unix systems, it returns $TMPDIR if non-empty, else /tmp. On Windows, it uses GetTempPath, returning the first non-empty value from %TMP%, %TEMP%, %USERPROFILE%, or the Windows directory.